Yurii_Khomych/l_3_oop/HW_3_yuliia/Functors/functors.py

```diff
@@ -0,0 +1,64 @@
+# example 1
+
+
+class ImproveEnglish:
+    def get_schedule(self, your_level):
+        if your_level in ['A1', 'A2']:
+            return self.schedule_a()
+
+        if your_level in ['B1', 'B2']:
+            return self.schedule_b()
+
+        if your_level in ['C1', 'C2']:
+            return self.schedule_c()
+
+    def schedule_a(self):
+        print('We have lessons in the morning, at 8:30 a.m')
+        return self
+
+    def schedule_b(self):
+        print("Unfortunately now we don't have groups align to your level")
+        return self
+
+    def schedule_c(self):
+        print('We have lessons in the evening, at 7:00 p.m')
+        return self
+
+
+you = ImproveEnglish()
+you.get_schedule('C1')
+
+# example 2
+
+
+class Profit:
+    def your_profit(self, salary, charges):
+        if charges == salary:
+            return self.find_job()
+        if charges > salary:
```

Comment:
`if,elif`...

Reply:
Corrected here https://github.com/YuriiKhomych/ITEA-advanced/pull/10 because of incorrect parent branch